server/athenian/api/internal/settings.py

```diff
@@ -838,11 +815,6 @@ async def set_release_matches(
                 .create_defaults()
                 .explode(with_primary_keys=True),
             )
-        if isinstance(self._sdb, Database):
-            sqlite = self._sdb.url.dialect == "sqlite"
```

Comment:
funny fact: flake8 was not complaining about unused `sqlite` due to `locals()` usage above

Reply:
Interesting